src/components/Map/Fabs/FacilityFilterFab/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect } from "react";
+import { Button } from "@nextui-org/react";
+import { getLocationCategories } from "../../../../constants/facility";
+import { useSearchParams } from "react-router-dom";
+import { getParamsFromDrawer } from "../../../../utils/routes/params";
+import { useGetClusterQuery } from "../../../../api/cluster";
+import FacilityMarker from "../../Markers/FacilityMarkers";
```

Comment:
```suggestion
import FacilityMarkers from "../../Markers/FacilityMarkers";
```

Seems to be a typo

Reply:
FacilityMarkers components remove from FacilityFilterFab !